FWCore/ParameterSet/python/MassReplace.py

```diff
@@ -38,7 +38,7 @@ def doIt(self,pset,base):
                             nrep = n; nrep.moduleLabel = self._paramReplace.moduleLabel
                             if self._verbose:print("Replace %s.%s[%d] %s ==> %s " % (base, name, i, n, nrep))
                             value[i] = nrep
-                elif isinstance(value, cms.InputTag):
+                elif value.isCompatibleCMSType(cms.InputTag):
                     if value == self._paramSearch:
```

Comment:
I suspect that something like ```eqWproxy``` from https://github.com/cms-sw/cmssw/pull/30071/files#diff-c2356aa3d79bc5ad0d7e89ea86cca856R12-R16 will be needed here, and apparently in the VInputTag case above

Reply:
this ```and value``` is more elegant